SAPHana/bin/SAPHanaSR-manageAttr

```diff
@@ -999,9 +999,13 @@ chk_cluster_health() {
     if [ -n "$ResID" ]; then
         res2chk="$ResID"
     fi
-    #rs=$(crm_mon -r1 2>/dev/null | awk '/Clone Set:/ { print $3 }')
+
+    # rs=$(crm_mon -r1 2>/dev/null | awk '/Clone Set:/ { print $3 }')
     # 'Clone Set:' or 'Master/Slave Set:' (on older pacemaker-cli versions)
-    rs=$(crm_mon -r1 2>/dev/null | awk '/[A-Za-z/ ][[:space:]]Set:/ { print $3 }')
+    # rs=$(crm_mon -r1 2>/dev/null | awk '/[A-Za-z/ ][[:space:]]Set:/ { print $3 }')
+    # if crm_mon output will change, awk's 'print $3' will no longer work correctly
+    # switch to cibadmin and xmllint instead.
+    rs=$(cibadmin -Q | xmllint -xpath "(//*[primitive[@*]]/@id)" - | sed -e 's/id=//g' -e 's/"//g')
```

Comment:
@angelabriel The idea is to serach for objects containing a primitive which have set at least one attribute and to return the object id. If you like to filter-out other clones or multi-state resources you might filter by the resource types:
`cibadmin -Q | xmllint -xpath '(//*[primitive[@type="SAPHanaTopology" or @type="SAPHanaController"]]/@id)' -`

Reply:
The origin idea from Lars was to check all clone or multi-stat resources for failcounts and warn the admin. But we can change this behaviour and shrink the check to 'our' resources. No problem, needs only a decision :-)

Reply:
So we release the PR for now and discuss that after vacation period ;-)